docs/api.md

```diff
@@ -307,6 +322,48 @@ If Google Chrome (rather than Chromium) is preferred, a [Chrome Canary](https://
 >
 > See [`this article`](https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/) for a description
   of the differences between Chromium and Chrome. [`This article`](https://chromium.googlesource.com/chromium/src/+/lkcr/docs/chromium_browser_vs_google_chrome.md) describes some differences for Linux users.
+  
+### class: Downloader
+
+A Downloader is manages downloaded chromium instances.
+
+Downloader is created per platform and operates revisions: strings representing 6-digit numbers, e.g. `"533271"`. Revisions could be checked with [omahaproxy.appspot.com](http://omahaproxy.appspot.com/).
+
+> **NOTE** Downloader class is not designed to work concurrently with other instances of Downloader class.
+
+#### downloader.canDownloadRevision(revision)
+- `revision` <[string]> a revision to check availability.
+- returns: <[Promise]<[boolean]>>  returns `true` if the revision could be downloaded from the host.
+
+The method initiates a HEAD request to check if revision is available.
+
+#### downloader.downloadRevision(revision[, progressCallback])
+- `revision` <[string]> a revision to download.
+- `progressCallback` <[function]> A function that will be called with two arguments:
+  - `downloadedBytes` <[number]> how many bytes are downloaded
+  - `totalBytes` <[number]> how many bytes is the total download
+- returns: <[Promise]> Resolves when the revision is downloaded and extracted
+
+The method initiates a GET request to download revision from the host. 
+
+#### downloader.downloadedRevisions()
+- returns: <[Promise]<[Array]<[string]>>> a list of all revisions downloaded for the given platform
```

Comment:
storedRevisions

A list of all revisions previously downloaded.

Reply:
Done